README.md

```diff
@@ -1,18 +1,19 @@
-[![](https://i.imgur.com/d0amtqs.png)](https://midlevel.github.io/MLAPI/)
+[![](https://i.imgur.com/d0amtqs.png)](https://mlapi.network/)
 
 [![GitHub Release](https://img.shields.io/github/release/MidLevel/MLAPI.svg?logo=github)](https://github.com/MidLevel/MLAPI/releases)
```

Comment:
Looks like this link has also changed, and redirects?
https://github.com/MidLevel/MLAPI/releases to https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/releases